src/vivarium_testing_utils/automated_validation/data_transformation/measures.py

```diff
@@ -355,10 +355,11 @@ def __init__(self, risk_factor: str) -> None:
             numerator=RiskStatePersonTime(risk_factor),
             denominator=RiskStatePersonTime(risk_factor, sum_all=True),
         )
+        self.risk_stratification_column = risk_factor
 
     @utils.check_io(data=SingleNumericColumn, out=SingleNumericColumn)
     def get_measure_data_from_sim_inputs(self, data: pd.DataFrame) -> pd.DataFrame:
-        return data
+        return data.rename_axis(index={"parameter": self.risk_stratification_column})
```

Comment:
Being consistent with renaming in relative risk

Reply:
This proved to be more work than I thought and it does seem a bit unnecessary since the risk stratification column will always be renamed to the risk factor.